src/features/unified-share-modal/ContactsField.js

```diff
@@ -28,7 +28,9 @@ type Props = {
     onContactAdd: Function,
     onContactRemove: Function,
     onInput?: Function,
+    onSuggestedCollaboratorAdd: Function,
```

Comment:
Can the `Function` be made specific? Like () => void or something

Reply:
I'll make this more specific.